diff --git a/board/toradex/verdin-imx8mp/verdin-imx8mp.c b/board/toradex/verdin-imx8mp/verdin-imx8mp.c
index e16841c3759..749fbdc78cc 100644
--- a/board/toradex/verdin-imx8mp/verdin-imx8mp.c
+++ b/board/toradex/verdin-imx8mp/verdin-imx8mp.c
@@ -219,6 +219,10 @@ int board_init(void)
 	init_usb_clk();
 #endif
 
+    /* turn vbat for eg91 - sodimm 206 - gpio1 / 0*/
+	gpio_request(IMX_GPIO_NR(1, 0), "eg91_vbat");
+	gpio_direction_output(IMX_GPIO_NR(1, 0), 1);
+
 	return ret;
 }
 
